tasks/gitea.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Create Gitea docker volumes
+  docker_volume:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - gitea_data
+    - gitea_mysql_data
+  register: volumes
+
+- name: Create MySQL container for Gitea
+  docker_container:
+    name: gitea-mysql
+    image: mysql:5.7
+    pull: true
+    volumes:
+      - gitea_mysql_data:/var/lib/mysql
+    env:
+      MYSQL_DATABASE: gitea
+      MYSQL_USER: gitea
+      MYSQL_PASSWORD: gitea
+      MYSQL_ROOT_PASSWORD: gitea
+    restart_policy: unless-stopped
+    memory: 1g
+
+- name: Create Gitea container
+  docker_container:
+    name: gitea
+    image: gitea/gitea:1.6
+    pull: true
+    links:
+        - gitea-mysql:db
+    volumes:
+      - gitea_data:/data

```

Comment:
Interested on why you've chosen volumes over bind mounts?

Reply:
No particular reason. I've done a bunch of reading on volumes vs bind mounts, and everything seems to be "use volumes, they are awesome", except if you want to access the content outside of the container (via, for example, a samba share :D).

With bind mounts, you get the permissions issues, but more flexibility. I'm happy to follow your standard of bind mounts, but I'd like to look at the permissions side of things. Similar to the way that FreeNAS jails handled it (create a app specific user on the host, give it permissions based on what it needs to access (create groups like movieusers or mediausers for example), and then pass in the UID/GID (where appropriate) to the container.

Thoughts?